ExampleTestProject/Assets/Testing/Tests/Client/EndpointTests.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace PlayFab.UUnit
 {
-    public class ConnectionStringTests : UUnitTestCase
+    public class Endpoint : UUnitTestCase
```

Comment:
I think this should have a "-Tests" suffix to visually differentiate from the new Endpoint rename